test/ex_doc/language/elixir_test.exs

```diff
@@ -450,7 +450,7 @@ defmodule ExDoc.Language.ElixirTest do
 
     test "Erlang stdlib types" do
       assert autolink_spec(quote(do: t() :: :sets.set())) ==
-               ~s[t() :: <a href="https://www.erlang.org/doc/man/sets.html#type-set">:sets.set</a>()]
+               ~s[t() :: <a href="https://www.erlang.org/doc/man/sets.html#t:set/0">:sets.set</a>()]
```

Comment:
✅ https://www.erlang.org/doc/man/sets.html#t:set/0 redirects to https://www.erlang.org/doc/apps/stdlib/sets.html#t:set/0